docs/installation.rst

```diff
@@ -68,12 +68,19 @@ Or download the `tarball`_:
 
     $ curl  -OL https://github.com/waynr/pyresume/tarball/master
 
-Once you have a copy of the source, you can install it with:
+Once you have a copy of the source, you can run a Docker image, which should
+bind mount PyResume source and a directory with YAML to convert:
 
 .. code-block:: console
 
-    $ python setup.py install
+    $ docker build --tag 'waynr/pyresume:latest' .
+    $ docker run --tty --interactive --volume /path/to/pyresume:/src/pyresume --volume /path/to/yaml/files:/mnt/pyresume waynr/pyresume:latest /bin/bash
+
+You can also install it directly with:
```

Comment:
I'm inclined to think that the primary recommended way to use `pyresume` should with python in the primary namespace. This is mostly because my intention is to eventually invoke docker within `pyresume` to execute texlive on the generated latex files. I would create a separate for all instructions related to the alternative `docker run` workflows.